### ISSUE TITLE
Do not translate is missing !

### ISSUE THREAD

**Issue by SyntheticMind** [ruby/rdoc#729]

**Comment by SyntheticMind** [comment]
I assuming that i am at the right place. I presume RDoc is used to create the class documentation.

There is a problem with the documentation. The "not translate" is not been uses on the names of classes and method.

ex: Welcome to the <span class="notranslate">Cool</span> company website!

As result, Google translate is changing the names of the classes and method. This simple change could provides a multilingual documentation for Ruby>

Thank you!

**Comment by zzak** [comment]
I don't think that is being added by RDoc:
https://github.com/ruby/rdoc/search?q=notranslate

Please re-open if I'm wrong. :pray:

**Comment by nobu** [comment]
I think we may be able to that attribute in the CSS files, if it is effective.

**Comment by SyntheticMind** [comment]
https://www.w3schools.com/tags/att_translate.asp

**Comment by SyntheticMind** [comment]
If you put this before your tag <code>, your example will be protected from
translation

**Comment by SyntheticMind** [comment]
Sorry!

Like that <code translate="no">

**Comment by aisayo** [comment]
Hi, is this still available to be worked on?

**Comment by olleolleolle** [comment]
@aisayo Yes, I believe this feature request _is_ available to be worked on.

[The MDN page about the HTML `translate` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/translate) is a helpful guide to the implementer.
